jpf-symbc/src/main/gov/nasa/jpf/symbc/SymbolicListener.java

```diff
@@ -228,16 +248,9 @@ public void instructionExecuted(VM vm, ThreadInfo currentThread, Instruction nex
             Config conf = vm.getConfig();
             String strInsn = executedInstruction.toString();
 
-            if(strInsn.contains("invokestatic") && strInsn.contains("Verifier.nondet")){
-                interceptSymbolic = true;
-            }
 
             if(!fileNameParsed){
```

Comment:
can you have another name than "fileNameParsed", it is not clear what is meant here. I suggest that the variable name indicates whether you have filled out the scope of the witness or not

Reply:
@sohah , I changed variable name to `witnessAssumptionScopeIsFilled`. Thank you for your advice!